wiki/fr/fr-Installation-for-Linux.md

```diff
@@ -0,0 +1,117 @@
+
+
+# Installation sous Linux
+
+Assurez-vous d'avoir déjà lu la page [Premier pas](Getting-Started) avant de commencer.
+
+**Pour l'instant, nous ne fournissons pas de paquets officiels pour Jamulus**
+
+Vous trouverez [ici]((https://github.com/corrados/jamulus/issues/223#issue-619038918){: target="_blank" rel="noopener noreferrer"}) une liste de paquets non officiels pour les différentes distributions. Vous pouvez également utiliser [ce script tout-en-un](Linux-Client-Install-Script) si vous le souhaitez.
+
+Cependant, la compilation des sources est assez facile :
+
+## Obtenir les sources de Jamulus
+
+1. Ouvrez une fenêtre de terminal (ligne de commande - `CTRL+ALT+t` sur Ubuntu et distributions connexes)
+1. Téléchargez et dézippez les sources de la dernière version :
+```shell
+wget https://github.com/corrados/jamulus/archive/latest.tar.gz
+tar -xvf latest.tar.gz
+```
+
+
+## Installation des dépendances
+
+Tout d'abord, mettez à jour la liste des paquets (par ex. sur les distributions basées sur Debian avec `sudo apt-get update`).
+
+Sur les distributions **basées sur Ubuntu** 18.04+, et sur **Debian** 9+ ou 10 et Raspberry Pi Raspbian Buster et suivantes :
+
+```shell
+sudo apt-get install build-essential qt5-qmake qtdeclarative5-dev qt5-default qttools5-dev-tools libjack-jackd2-dev
+```
+
+Sur **Fedora**:
+
+```shell
+sudo dnf install qt5-qtdeclarative-devel jack-audio-connection-kit-dbus jack-audio-connection-kit-devel
+```
+
+### Qjackctl : Optionnel, mais recommandé
+
+[QjackCtl](https://qjackctl.sourceforge.io) est un utilitaire qui vous aide à configurer le serveur audio Jack (installé dans le cadre des dépendances ci-dessus). Installez-le, par ex., via
+
+```shell
+sudo apt-get install qjackctl
+```
+
+Vous pouvez également envisager d'utiliser un [noyau à faible latence](https://help.ubuntu.com/community/UbuntuStudio/RealTimeKernel) (ex. pour Ubuntu 18.04 : `sudo apt-get install linux-lowlatency-hwe-18.04`).
+
+
+## Compilons le sale gosse
+
+Maintenant, `cd` dans le répertoire des sources de Jamulus que avez téléchargé :
+
+```shell
+cd jamulus-latest
+```
+Maintenant, compilez les sources avec les commandes suivantes (le dernier `make` peut prendre plusieurs minutes à s'exécuter) :
+
+```shell
+qmake Jamulus.pro
+make clean
+make
+sudo make install
+```
+
+
+## Configurez votre interface audio
+
+### Configurer Jack avec QJackCtl
+
+Les clients Jamulus ont besoin de [Jack](https://jackaudio.org/){: target="_blank" rel="noopener noreferrer"} pour fonctionner, mais vous devez d'abord le configurer. La méthode recommandée est d'utiliser `QjackCtl`.
+
+1. Ouvrez le terminal, par ex. avec Crtl-Alt-T et
+1. Exécutez la commande `qjackctl` vous verrez le **Jack Audio Connection Kit**
+2. Configurez votre interface audio comme suit (les paramètres exacts pour Jack dépendront des fonctionnalités de votre interface audio/carte son) :
+
+- Réglez l'**Interface** sur celle que vous souhaitez (il peut y en avoir plusieurs dans la liste)
```

Comment:
I had used "Interface" without audio because on the UI of qjackctl that is "Interface" only.
But "Interface audio" is fine for me (better than "Interface" only)

<img width="855" alt="Capture d’écran 2020-11-29 à 18 02 24" src="https://user-images.githubusercontent.com/9108457/100548478-1a5c1b00-326d-11eb-83f8-3eaee13a2ea1.png">

Reply:
ok

Reply:
We keep "Interface audio"? Or we use "Interface"?